icebox/Makefile

```diff
@@ -36,45 +36,66 @@ clean:
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
-	cp chipdb-384.txt    $(DESTDIR)$(PREFIX)/share/icebox/
-	cp chipdb-1k.txt     $(DESTDIR)$(PREFIX)/share/icebox/
-	cp chipdb-8k.txt     $(DESTDIR)$(PREFIX)/share/icebox/
-	cp chipdb-5k.txt     $(DESTDIR)$(PREFIX)/share/icebox/
-	cp chipdb-u4k.txt    $(DESTDIR)$(PREFIX)/share/icebox/
-	cp chipdb-lm4k.txt   $(DESTDIR)$(PREFIX)/share/icebox/
-	cp icebox.py         $(DESTDIR)$(PREFIX)/bin/icebox.py
-	cp iceboxdb.py       $(DESTDIR)$(PREFIX)/bin/iceboxdb.py
-	cp icebox_chipdb.py  $(DESTDIR)$(PREFIX)/bin/icebox_chipdb$(PY_EXE)
-	cp icebox_diff.py    $(DESTDIR)$(PREFIX)/bin/icebox_diff$(PY_EXE)
-	cp icebox_explain.py $(DESTDIR)$(PREFIX)/bin/icebox_explain$(PY_EXE)
-	cp icebox_asc2hlc.py $(DESTDIR)$(PREFIX)/bin/icebox_asc2hlc$(PY_EXE)
-	cp icebox_hlc2asc.py $(DESTDIR)$(PREFIX)/bin/icebox_hlc2asc$(PY_EXE)
-	cp icebox_colbuf.py  $(DESTDIR)$(PREFIX)/bin/icebox_colbuf$(PY_EXE)
-	cp icebox_html.py    $(DESTDIR)$(PREFIX)/bin/icebox_html$(PY_EXE)
-	cp icebox_maps.py    $(DESTDIR)$(PREFIX)/bin/icebox_maps$(PY_EXE)
-	cp icebox_vlog.py    $(DESTDIR)$(PREFIX)/bin/icebox_vlog$(PY_EXE)
-	cp icebox_stat.py    $(DESTDIR)$(PREFIX)/bin/icebox_stat$(PY_EXE)
+	cp chipdb-384.txt    $(DESTDIR)$(PREFIX)/share/$(PROGRAM_PREFIX)icebox/
+	cp chipdb-1k.txt     $(DESTDIR)$(PREFIX)/share/$(PROGRAM_PREFIX)icebox/
+	cp chipdb-8k.txt     $(DESTDIR)$(PREFIX)/share/$(PROGRAM_PREFIX)icebox/
+	cp chipdb-5k.txt     $(DESTDIR)$(PREFIX)/share/$(PROGRAM_PREFIX)icebox/
+	cp chipdb-u4k.txt    $(DESTDIR)$(PREFIX)/share/$(PROGRAM_PREFIX)icebox/
+	cp chipdb-lm4k.txt   $(DESTDIR)$(PREFIX)/share/$(PROGRAM_PREFIX)icebox/
+	cp icebox.py         $(DESTDIR)$(PREFIX)/bin/$(subst -,_,$(PROGRAM_PREFIX))icebox.py
+	cp iceboxdb.py       $(DESTDIR)$(PREFIX)/bin/$(subst -,_,$(PROGRAM_PREFIX))iceboxdb.py
+	cp icebox_chipdb.py  $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_chipdb$(PY_EXE)
+	cp icebox_diff.py    $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_diff$(PY_EXE)
+	cp icebox_explain.py $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_explain$(PY_EXE)
+	cp icebox_asc2hlc.py $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_asc2hlc$(PY_EXE)
+	cp icebox_hlc2asc.py $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_hlc2asc$(PY_EXE)
+	cp icebox_colbuf.py  $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_colbuf$(PY_EXE)
+	cp icebox_html.py    $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_html$(PY_EXE)
+	cp icebox_maps.py    $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_maps$(PY_EXE)
+	cp icebox_vlog.py    $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_vlog$(PY_EXE)
+	cp icebox_stat.py    $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_stat$(PY_EXE)
+	sed -i 's+import iceboxdb+import $(subst -,_,$(PROGRAM_PREFIX))iceboxdb as iceboxdb+g' $(DESTDIR)$(PREFIX)/bin/$(subst -,_,$(PROGRAM_PREFIX))icebox.py
+	sed -i 's+import icebox+import $(subst -,_,$(PROGRAM_PREFIX))icebox as icebox+g' $(PREFIX)/bin/$(PROGRAM_PREFIX)icebox_chipdb$(PY_EXE)
```

Comment:
@mmicko, @cliffordwolf, I believe there is a bug here, since `icebox_chipdb` is missing `DESTDIR`. I found this while trying to execute https://github.com/ghdl/docker/blob/master/dockerfiles/cache_pnr#L35-L55. See https://github.com/ghdl/docker/runs/577351652?check_suite_focus=true#step:3:688

Reply:
Thanks for spotting this.

Reply:
Kudos to continuous integration 😆 (ghdl/docker#27)